core/src/main/java/org/apache/accumulo/core/metadata/schema/TabletMetadata.java

```diff
@@ -465,6 +459,9 @@ public static <E extends Entry<Key,Value>> TabletMetadata convertRow(Iterator<E>
     while (rowIter.hasNext()) {
       final Entry<Key,Value> kv = rowIter.next();
       final Key key = kv.getKey();
+      if (key.isDeleted()) {
```

Comment:
When did you see this?  Should not see deletes during scan.

Reply:
I think it was in one of the tests where I was inserting the nextOperationId and then deleting it. The test never entry being deleted and it kept hanging. I'm going to remove this and see if everything is good now.

Reply:
I removed this and ManagerAssignmentIT still passes.